scss/mixins/_banner.scss

```diff
@@ -0,0 +1,10 @@
+@mixin bsBanner($file, $suffix:"") {
+  /*!
+   * Bootstrap #{$file} v5.2.0-beta1 (https://getbootstrap.com/)
+   * Copyright 2011-2022 The Bootstrap Authors
+   * Copyright 2011-2022 Twitter, Inc.
+   * Licensed under MIT (https://github.com/twbs/bootstrap/blob/main/LICENSE)
+   * #{$suffix}
```

Comment:
I've tried to remove this line when `$suffix` is empty but haven't succeeded. IDK how to remove a line or add it dynamically in this context.
Tried things in this spirit: `* Licensed under MIT (https://github.com/twbs/bootstrap/blob/main/LICENSE)#{if($suffix!='',\r\n'  * ' + $suffix,null)}`.

Reply:
Yep I understand. My picky side has been triggered by it so I tried to find a solution 😄 
I resolve both comments and approve the PR